backend/etl/processor.js

```diff
@@ -130,10 +132,62 @@ class EtlJobProcessor {
     return Boolean(published)
   }
 
-  _customDomain(shop) {
-    // TODO(franck): figure out how to extract this information.
-    log.warn(`Shop ${shop.id}: custom domain extraction not implemented yet.`)
-    return null
+  // Check if the most recent publish (if any) used a custom domain.
+  async _customDomain(shop) {
+    const custom = await ShopDomain.findOne({
+      where: { shopId: shop.id },
+      order: [['id', 'desc']]
+    })
+    return custom ? custom.domain.indexOf('ogn.app') > 0 : false
```

Comment:
I don't really think it has any benefit here, but you can use ILIKE like this with Sequelize:

    where: { domain: { [Op.iLike]: '%ogn.app' } }

No need to change.

Reply:
Ah yeah. Good to know. Thanks.